### DISCUSSION TITLE
Downsized to necessary basics version

### DISCUSSION THREAD

**Discussion by mukral** [orestbida/cookieconsent#459] [General]
Hi,
is it possible to downsize to a just working minimum version?
I'm working on a project where it's not that easy (at least for me) to integrate a complete css-file (complete page building, design and running the page is directly done in/from python, it's possible to run javascript, but design is directly bound to the objects ( I use nicegui - best ever).
What I've experienced in my testings is that Cookieconsent only works (or the frame is shown) if there a css-file is loaded before.
For my understanding css is for design reasons. For the beginning I don't need all the design.

It's so hard for me to understand what happens and how it is bound to each other. It would be very helpful if there is a version that just works. Desing comes later... Would it be possible (and nice for other people) to strip it down or to have a how-to for this?

I know I'm a newbie and it would be also helpful for the learning curve - I also know this is all basic stuff and I will read the tutorials, but this will need months...

Thank you
Markus

**Comment by orestbida** [comment]
Hi @mukral,

I think the out-of the box UI is the main reason why someone might choose this plugin over others. This is even more true for the [upcoming v3](https://playground.cookieconsent.orestbida.com/). I won't be working on a stripped down version of the current UI, as I don't see myself using it.

I'm not sure how `NiceGUI` works, but it shouldn't be that hard to include an external stylesheet. If your goal is to create an app with a unified design language, your best bet would be to create an ad-hoc solution based on your needs, sorry.
